sdk/webpubsub/azure-messaging-webpubsub-client/src/main/java/com/azure/messaging/webpubsub/client/WebPubSubAsyncClient.java

```diff
@@ -1086,27 +1035,26 @@ WebSocketSession getWebsocketSession() {
 
     private Sinks.EmitFailureHandler emitFailureHandler(String message) {
         return (signalType, emitResult) -> {
-            LoggingUtils.addSignalTypeAndResult(this.logger.atWarning(), signalType, emitResult)
-                .log(message);
+            LoggingUtils.addSignalTypeAndResult(this.logger.atWarning(), signalType, emitResult).log(message);
             return emitResult.equals(Sinks.EmitResult.FAIL_NON_SERIALIZED);
         };
     }
 
-    private RuntimeException logSendMessageFailedException(
-        String errorMessage, Throwable cause, boolean isTransient, WebPubSubMessage message) {
+    private RuntimeException logSendMessageFailedException(String errorMessage, Throwable cause, boolean isTransient,
+        WebPubSubMessage message) {
 
         return logSendMessageFailedException(errorMessage, cause, isTransient,
             (message instanceof WebPubSubMessageAck) ? ((WebPubSubMessageAck) message).getAckId() : null);
     }
 
-    private RuntimeException logSendMessageFailedException(
-        String errorMessage, Throwable cause, boolean isTransient, Long ackId) {
+    private RuntimeException logSendMessageFailedException(String errorMessage, Throwable cause, boolean isTransient,
+        Long ackId) {
 
         return logSendMessageFailedException(errorMessage, cause, isTransient, ackId, null);
     }
 
-    private RuntimeException logSendMessageFailedException(
-        String errorMessage, Throwable cause, boolean isTransient, Long ackId, AckResponseError error) {
+    private RuntimeException logSendMessageFailedException(String errorMessage, Throwable cause, boolean isTransient,
```

Comment:
There are many changes in this file, are these all formatting changes, or is there code change?

Reply:
This was just the Eclipse formatting rules getting applied to this file. I can revert them if wanted